gene_finder.py

```diff
@@ -2,37 +2,48 @@
 """
 YOUR HEADER COMMENT HERE
```

Comment:
For next mini project, please write a short description about the code (header comment really helps when your code base gets large)